packages/artillery-plugin-publish-metrics/test/config-newrelic-api.yaml

```diff
@@ -7,9 +7,14 @@ config:
     publish-metrics:
       - type: newrelic
         region: 'us'
-        licenseKey: "{{ $processEnvironment.NR_LICENSE_KEY }}"
+        licenseKey: '{{ $processEnvironment.NR_LICENSE_KEY }}'
```

Comment:
Any particular reason to change double quotes to single quotes here?

Reply:
Unfortunately It seems I had forgotten about this file when submitting the formatting changes to New Relic reporter prior to this PR. My local formatting has been set to match the prettier configuration found in Artillery's code, and it had changed the quotes automatically to single quotes. Since I only noticed after I made the PR I left it as is considering that it had inconsistent formatting before with both single and double quotes. 
I should of left a comment explaining this though.